src/isomorphic/ReactDebugTool.js

```diff
@@ -58,17 +58,38 @@ var ReactDebugTool = {
   onSetState() {
     emitEvent('onSetState');
   },
-  onMountRootComponent(internalInstance) {
-    emitEvent('onMountRootComponent', internalInstance);
+  onSetIsTopLevelWrapper(debugID, isTopLevelWrapper) {
```

Comment:
This information is an implementation detail that I'm trying to get rid of ASAP. It shouldn't be of interest to a debug tool to know this.

Can you just bypass it and emit the child of this as the conceptual root?